primitives/application-crypto/test/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
```

Comment:
Instead of creating `src/lib.rs` you could just create `tests` folder and put every module in it's own file (`ed25519.rs`, `sr25519.rs`) and make this crate purely a test crate.